server/src/internalClusterTest/java/org/elasticsearch/search/query/SearchQueryIT.java

```diff
@@ -2079,4 +2079,21 @@ public void testIssueFuzzyInsideSpanMulti() {
         SearchResponse response = client().prepareSearch("test").setQuery(query).get();
         assertHitCount(response, 1);
     }
+
+    public void testFetchIdFieldQuery() {
+        createIndex("test");
+        int docCount = randomIntBetween(10, 50);
+        for (int i=0; i<docCount; i++) {
```

Comment:
nit: minor formatting which will eventually block the CI checks, run `./gradlew  :server:spotlessApply` to correct this here and in some other places.